test/test.js

```diff
@@ -32,7 +32,8 @@ const VALID_MEASUREMENT = {
   carChecksum: 'somehash',
   minerId: 'f02abc',
   providerId: 'provider-pubkey',
-  indexerResult: 'OK'
+  indexerResult: 'OK',
+  stationId: '123e4567-e89b-12d3-a456-426614174000'
```

Comment:
IIRC, we are not going to use GUIDs for station IDs. Let's use a different value in the test data then, e.g. `some-station-id`.

Reply:
Ah yes, remnant of when I was building this with UUID in mind, good catch!